src/libman/libman.nuspec

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
   <metadata>
-    <id>libman</id>
+    <id>Microsoft.Web.LibraryManager.Tools</id>
     <tags>libman</tags>
```

Comment:
>libman [](start = 10, length = 6)

Intentionally not changed?

Reply:
Correct, we want to keep `libman` as a tag to help when searching.
We should potentially add more tags like `LibraryManager` , `LibraryManagerCLI`